aptos-move/block-executor/src/executor.rs

```diff
@@ -95,8 +98,8 @@ where
         }
     }
 
-    fn fallback_to_sequential() {
-        unimplemented!();
+    fn fallback_to_sequential<M: std::fmt::Debug>(message: M) {
+        panic!("{:?}", message);
```

Comment:
nit: I think you can add a message to unimplemented, and that panics

Reply:
this is anyways changed later to correct thing